addons/binding/org.openhab.binding.sinope/readme.md

```diff
@@ -0,0 +1,145 @@
+
+This binding supports multiple gateways with multiple devices.
+
+---
+
+## Supported Things
+
+The Sinopé bridge is required as a "bridge" for accessing any other Sinopé devices.
+
+Right now, only the thermostat devices (3000W and 4000W) (TH1120RF) are supported.
+
+## Discovery
+
+The Sinopé Gateway (bridge) discovery is not supported for now. It will be added in future release.
+The Sinopé devices discovery is implemented. 
+
+## Prerequisites
+
+### Bridge or the Sinopé Gateway 
+First, you will need to get your API key from your Sinopé gateway.
+
+Grab the latest release of the [sinope-core library](<https://github.com/chaton78/sinope-core/releases>)
+
+On Windows, you can run the SinopeProtocol.exe (in the zip release). 
+The gateway parameter is written on the back of the SinopéGateway (example, 002f-c2c2-dd88-aaaa). 
+The addr parameter is the IP given to your gateway.
+
+```
+SinopeProtocol.exe -addr [YOUR_GATEWAY_IP_OR_HOSTNAME]  -gateway [YOUR_GATEWAY_ID] -login
+Getting API Key  - PRESS WEB Button
+Your api Key is: 0x12 0x57 0x55 0xD5 0xCD 0x4A 0xD5 0x33
+
+```
+ On other operating systems, using only a JVM, you can invoke directly the java command from the latest release of the [sinope-core library](<https://github.com/chaton78/sinope-core/releases>):
+ 
+```
+java -jar core-0.0.3-shaded.jar -addr [YOUR_GATEWAY_IP_OR_HOSTNAME]   -gateway [YOUR_GATEWAY_ID] -login
+Getting API Key  - PRESS WEB Button
+Your api Key is: 0x12 0x57 0x55 0xD5 0xCD 0x4A 0xD5 0x33
+
+```
+### Thing or device discovery
+
+You can use the same procedure to discover each device you want to use. 
+You will need to provide the api key from the previous step. 
+If you use spaces, please, use double quotes to pass the api key (i.e. "0x12 0x57 0x55 0xD5 0xCD 0x4A 0xD5 0x33") 
+
+Use the device procedure to discover it. 
+For a thermostat, you need to push both buttons. 
+The application will loop forever, press ctrl-c to exit.
+
+```
+SinopeProtocol.exe -addr [YOUR_GATEWAY_IP_OR_HOSTNAME]  -gateway [YOUR_GATEWAY_ID] -api "[YOUR_API_KEY]" -discover
+
+It is now time to push both buttons on your device!
+Press crtl-c to exit!
+Your device id is: 0x00 0x00 0x35 0x86
+It is now time to push both buttons on your device!
+Press crtl-c to exit!
+```
+On other operating systems, using only a JVM, you can invoke directly the java command:
+
+```
+java -jar core-0.0.3-shaded.jar -addr [YOUR_GATEWAY_IP_OR_HOSTNAME]  -gateway [YOUR_GATEWAY_ID] -api "[YOUR_API_KEY]" -discover
+
+It is now time to push both buttons on your device!
+Press crtl-c to exit!
+Your device id is: 0x00 0x00 0x35 0x86
+It is now time to push both buttons on your device!
+Press crtl-c to exit!
+```
+## Thing Configuration
+
+The Sinopé bridge requires the address, the gateway id and the API key in order for the binding to know where and how to access it.
+In the thing file, this looks e.g. like
```

Comment:
This should be part of the "Full Example" only. Better add a table here that describes the parameters (as they can also be set in the Paper UI or by other means).